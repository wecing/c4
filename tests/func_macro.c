#define F(x) +1

extern int printf(char *s, ...);

int main() {
  int F = 42;
  printf("42 = %d\n", F);
  printf("42 + 1 = %d\n", F F(_));
  return 0;
}
