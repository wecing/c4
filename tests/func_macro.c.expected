42 = 42
42 + 1 = 43
